src/css/pages/resources.css

```diff
@@ -24,4 +24,16 @@
     margin-top: 25px;
 
     font-size: 10px;
+}
+
+.resource-filters {
+    padding-bottom: 25px;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-column-gap: 15px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5692903)